django/core/management/commands/inspectdb.py

```diff
@@ -322,6 +319,13 @@ def normalize_col_name(self, col_name, used_column_names, is_relation):
 
         return new_name, field_params, field_notes
 
+    @staticmethod
+    def table_name_to_model(table_name):
```

Comment:
This is a new method intended for an easier customization so we should avoid using `staticmethod` here.